<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>McoAVPlay01Class</class>
 <widget class="QWidget" name="McoAVPlay01Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>McoAVPlay01</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="McoGLWidget" name="openGLWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="playBtn">
     <property name="text">
      <string>开始播放</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="continueBtn">
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="suspendBtn">
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="screenShotBtn">
     <property name="text">
      <string>截图</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="openBtn">
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>McoGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>mcoglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="McoAVPlay01.qrc"/>
 </resources>
 <connections/>
</ui>
